iothub_client/inc/iothub_client_core_common.h

```diff
@@ -150,14 +150,80 @@ extern "C"
 
     MU_DEFINE_ENUM_WITHOUT_INVALID(DEVICE_TWIN_UPDATE_STATE, DEVICE_TWIN_UPDATE_STATE_VALUES);
 
+/**
+ * @brief Current versions of @p IOTHUB_CLIENT_GET_TWIN_REQUEST_OPTIONS structure.
+ */
+#define IOTHUB_CLIENT_GET_TWIN_REQUEST_OPTIONS_STRUCT_VERSION_1 1
+
+/**
+ * @brief   GET twin request options
+ */
+typedef struct IOTHUB_CLIENT_GET_TWIN_REQUEST_OPTIONS_TAG
+{
+    /**
+     * @brief   Version of the structure.
+     *
+     * @note    Currently must be @p IOTHUB_CLIENT_GET_TWIN_REQUEST_OPTIONS_STRUCT_VERSION_1.
```

Comment:
```suggestion
     * @note    Must be @p IOTHUB_CLIENT_GET_TWIN_REQUEST_OPTIONS_STRUCT_VERSION_1.
```

Reply:
trying to stay consistent with what is in PnP. https://github.com/Azure/azure-iot-sdk-c/blob/01f0d50e23fa2a8dbd16f4e3508fa18acce7e5c0/iothub_client/inc/iothub_client_properties.h#L127